src/Core/Components/NavMenu/FluentNavMenu.razor

```diff
@@ -1,39 +1,45 @@
 ﻿@namespace Microsoft.Fast.Components.FluentUI
 @inherits FluentComponentBase
 
-<div id="@Id">
-    <CascadingValue Value="this" IsFixed="true">
-        <CascadingValue TValue=INavMenuItemsOwner Value="@this" IsFixed>
-            <CascadingValue Value="@Expanded" Name="NavMenuExpanded">
-                <CascadingValue Value="@_hasChildIcons" Name="NavMenuItemSiblingHasIcon">
-                    <FluentTreeView Class="@ClassValue"
-                                    Style="@StyleValue"
-                                    CurrentSelected="@_currentlySelectedTreeItem"
-                                    CurrentSelectedChanged="@HandleCurrentSelectedChangedAsync">
-                        @if (Collapsible)
+
+@{
+    string? title;
+    if (Collapsible)
+    {
+        title = null;
+    }
+    else 
+{
+        title = Title;
+    }
+}
+
+<div id="@Id" @attributes="AdditionalAttributes" class="@ClassValue" style="@StyleValue" aria-label="@title">
```

Comment:
Could you set `aria-label="@(Collapsible ? null : title)"` and remove the code above?